embeds/nextgen17/style.scss

```diff
@@ -0,0 +1,175 @@
+// Default breakpoints from Frontend
+$mq-breakpoints: (
+    mobile:          320px,
+    mobileLandscape: 480px,
+    phablet:         660px,
+    tablet:          740px,
+    desktop:         980px,
+    leftCol:         1140px,
+    wide:            1300px
+);
+
+$pink: #ec2991;
+$green: #aad801;
+$bg: #2A2A2A;
+
+@import 'shared/_imports';
+
+#nextgen17.fc-container--thrasher {
```

Comment:
This mixin can do this bit for you, plus a few other bits:

https://github.com/guardian/thrashers/blob/master/shared/_mixins.scss#L3-L19